src/audio/kpb.c

```diff
@@ -285,34 +289,36 @@ static int kpb_bind(struct comp_dev *dev, void *data)
 	bu = (struct ipc4_module_bind_unbind *)data;
 	buf_id = IPC4_COMP_ID(bu->extension.r.src_queue, bu->extension.r.dst_queue);
 
-	/* We're assuming here that KPB Real Time sink (kpb->sel_sink) is
+	/* We're assuming here that KPB Real Time sink (kpb->sinks[0]) is
```

Comment:
does this assumption depend on the topology? So it's possible to construct a "wrong" topology where this won't be true but that otherwise will be valid?